tests/foreman/ui/test_role.py

```diff
@@ -199,12 +216,46 @@ def test_positive_create_non_overridable_filter(self):
 
         1. Filter is created without taxonomies
         2. Override checkbox is not available to check.
-        3. Filter doesnt inherits taxonomies from role
-
-        @caseautomation: notautomated
         """
+        name = gen_string('alpha')
+        username = gen_string('alpha')
+        password = gen_string('alpha')
+        with Session(self.browser) as session:
+            make_role(
+                session,
+                name=name,
+                locations=[self.role_loc],
+                organizations=[self.role_org],
+            )
+            self.assertIsNotNone(self.role.search(name))
+            self.role.add_permission(
+                name,
+                resource_type='Architecture',
+                permission_list=['view_architectures', 'edit_architectures'],
+                override_check=False,
+            )
+            self.assertTrue(self.role.wait_until_element(
+                common_locators['alert.success']))
+            make_user(
+                session,
+                username=username,
+                password1=password,
+                password2=password,
+                roles=[name],
+                locations=[self.role_loc],
+                organizations=[self.role_org],
+                edit=True
+            )
+        with Session(self.browser, username, password) as session:
+            set_context(session, org=self.role_org)
+            set_context(session, loc=self.role_loc)
+            self.assertIsNone(session.nav.wait_until_element(
+                menu_locators['menu.content'], timeout=3))
+            self.assertIsNone(session.nav.wait_until_element(
+                menu_locators['menu.infrastructure'], timeout=3))
+            # check that we can access edit functionality at all
+            self.architecture.update(old_name='x86_64', new_name='x86_64')
 
```

Comment:
In this Testcase you would have check 
1. if the locations and organizations tab is not present in Architecture filter.
2. If the override checkbox is not present in the Architecture filter.